Sources/Views/NewEpisodePage.swift

```diff
@@ -116,7 +116,7 @@ private func sideBar(
           Class.border.all,
           Class.border.rounded.all,
           Class.pf.colors.border.gray850,
-          Class.margin([.mobile: [.left: 3]])
+          Class.margin([.mobile: [.leftRight: 2], .desktop: [.left: 3, .right: 0]])
```

Comment:
the margin that I had before was only appropriate for desktop, not mobile. on mobile we need both left and right margins for the side bar